src/components/todos/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
try to add `aria-label` attribute to buttons all over the project to avoid using `eslint-disable`.
More information  [here]( https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/control-has-associated-label.md)